mobile/src/lib/filesystem.js

```diff
@@ -0,0 +1,85 @@
+const ERROR_GET_DIRECTORY = 'Error to get directory'
+const ERROR_WRITE_FILE = 'Error to write file'
+const ERROR_GET_FILE = 'Error to get file'
+const ERROR_GET_COZY_ENTRY = 'Error to get cozy entry'
+
+export const getRootPath = () => {
+  return window.cordova.platformId === 'android'
+    ? window.cordova.file.externalRootDirectory
+    : window.cordova.file.dataDirectory
+}
+
+export const getCozyPath = () => {
+  return 'Cozy/Cozy Files/'
+}
+
+export const getEntry = (path) => {
+  return new Promise((resolve, reject) => {
+    window.resolveLocalFileSystemURL(path, resolve, reject)
+  })
+}
+
+export const getCozyEntry = () => {
+  return new Promise((resolve, reject) => {
+    getEntry(getRootPath() + getCozyPath())
+    .then(resolve)
+    .catch(
+      getEntry(getRootPath())
+      .then(entry => {
+        getDirectory(entry, 'Cozy').then(entry => {
+          getDirectory(entry, 'Cozy Files').then(resolve).catch(reject)
+        }).catch(reject)
```

Comment:
And maybe it worth to get strings `'Cozy'` and `'Cozy Files'` from the `getCozyPath()` to have DRY respected.